Modules/TOF/src/TOFCheckCompressedCounter.cxx

```diff
@@ -0,0 +1,98 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   TOFCheckCompressedCounter.cxx
+/// \author Nicolo' Jacazio
+///
+
+// QC
+#include "TOF/TOFCheckCompressedCounter.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+
+// ROOT
+#include <TH1.h>
+#include <TH2.h>
+#include <TPaveText.h>
+#include <TList.h>
+
+using namespace std;
+
+namespace o2::quality_control_modules::tof
+{
+
+void TOFCheckCompressedCounter::configure(std::string) {}
+
+Quality TOFCheckCompressedCounter::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
+{
+
+  Quality result = Quality::Null;
+  ILOG(Info) << "Checking quality of diagnostic histogram" << ENDM;
+
+  for (auto& [moName, mo] : *moMap) {
+    (void)moName;
+    if (mo->getName() == "RDHCounterCrate0") {
+      auto* h = dynamic_cast<TH1F*>(mo->getObject());
+      if (h->GetEntries() == 0) {
+        result = Quality::Medium;
+      }
+    }
+  }
+  return result;
+}
+
+std::string TOFCheckCompressedCounter::getAcceptedType() { return "TH1F"; }
+
+void TOFCheckCompressedCounter::beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult)
```

Comment:
Have you retested the beautify since v0.23 ?

Reply:
No, not yet, I will try to recompile with latest master and see whether it has an effect

Reply:
I will probably do this asap the PR is merged and I will recompile everything from scratch!
Marking this as resolved so far!